common/common_user_bpf_xdp.c

```diff
@@ -244,3 +230,40 @@ int check_map_fd_info(int map_fd, struct bpf_map_info *info,
 
 	return 0;
 }
+
+int open_bpf_map_file(const char *pin_dir,
+		      const char *mapname,
+		      struct bpf_map_info *info)
+{
+	char filename[PATH_MAX];
+	int err, len, fd;
+	__u32 info_len = sizeof(*info);
+
+	len = snprintf(filename, PATH_MAX, "%s/%s", pin_dir, mapname);
+	if (len < 0) {
+		fprintf(stderr, "ERR: constructing full mapname path\n");
+		return -1;
+	}
+
+	/* Lesson#1: There is only a weak dependency to libbpf here as
```

Comment:
Not sure if this comment makes sense anymore. Certainly the lession#1 part should go :)

Reply:
@tohojo I removed this comment, as other functions in the file do depend on `libbpf`

Reply:
OK, SGTM. I don't see that in the PR, though, did you push your updated branch?

Reply:
@tohojo thank, just pushed